OpenTabletDriver.Configurations/Configurations/XP-Pen/Deco 01 V2 V2.json

```
@@ -0,0 +1,43 @@
+{
+  "Name": "XP-Pen Deco 01 V2 V2",
```

I really don't know how to feel about this name, might be better to call it something like `XP-Pen Deco 01 V2 Rev B` or anything that could help differentiate it. Really annoying that manufacturers do this an don't update their model names.

---

Should probably change scheme to `Tablet Model (variant 1)`

Since these are not guaranteed a revision, nor a version, unless specified by OEM. This scheme rolls off the tongue much better too for this tablet.

`XP-Pen Deco 01 V2 (variant 1)`
`XP-Pen Deco 01 V2 (variant 2)`

We may hide variant name from UX.

Or just integrate spec variants directly in configs. Depending on traction here we'll raise this to a new feature issue.

---

For now I'm in favor of referring to it as such 
```suggestion
  "Name": "XP-Pen Deco 01 V2 (variant 2)",
```
TODO in the future to add a better system for this variant issue.